alpha-commons/src/main/java/at/ac/tuwien/kr/alpha/commons/programs/reification/ReificationContext.java

```diff
@@ -0,0 +1,54 @@
+package at.ac.tuwien.kr.alpha.commons.programs.reification;
+
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+import at.ac.tuwien.kr.alpha.api.programs.Predicate;
+import at.ac.tuwien.kr.alpha.api.programs.atoms.BasicAtom;
+import at.ac.tuwien.kr.alpha.api.programs.terms.ConstantTerm;
+import at.ac.tuwien.kr.alpha.commons.programs.atoms.Atoms;
+import at.ac.tuwien.kr.alpha.commons.programs.terms.Terms;
+import at.ac.tuwien.kr.alpha.commons.util.IdGenerator;
+
+class ReificationContext implements IdGenerator<ConstantTerm<?>>{
```

Comment:
```suggestion
class ReificationContext implements IdGenerator<ConstantTerm<?>> {
```
Checkstyle complains otherwise.